### DISCUSSION TITLE
Rubberduck Folder Annotations

### DISCUSSION THREAD

**Discussion by MarkJohnstoneGitHub** [GCuser99/SeleniumVBA#54] [Ideas]
Is it possible to add Rubberduck folder annotations to  at least separate the testing modules?  It would also handy for the Selenium webdriver classes as well when viewing in the RD Code Explorer.   See https://rubberduckvba.com/Annotations/Details/Folder 

Eg.
'Rubberduck annotations
'@Folder("SeleniumVBA.Testing")

**Comment by GCuser99** [comment]
Will do.
